meta-refkit/classes/image-dsk.bbclass

```diff
@@ -227,6 +227,16 @@ python do_uefiapp_setscene () {
     sstate_setscene(d)
 }
 
+uefiapp_sign() {
+    if [ -e "${REFKIT_DB_KEY}" -a -e "${REFKIT_DB_CERT}" ] ; then
+        for i in `find ${DEPLOYDIR} -name '*.efi'`; do
+            sbsign --key ${REFKIT_DB_KEY} --cert ${REFKIT_DB_CERT} $i
+            sbverify --cert ${REFKIT_DB_CERT} $i.signed
+            mv $i.signed $i
+        done
+    fi
+}
```

Comment:
I've added ```do_rootfs[vardeps] += "REFKIT_DB_CERT REFKIT_DB_KEY"``` and a comment mentioning that it guarantees rebuilding/re-signing if the filename changes.